Storage/Storage/Model/Site+CoreDataProperties.swift

```diff
@@ -22,6 +22,7 @@ extension Site {
     @NSManaged public var isJetpackConnected: Bool
     @NSManaged public var isJetpackThePluginInstalled: Bool
     @NSManaged public var jetpackConnectionActivePlugins: [String]?
+    @NSManaged public var isBlogPublic: Bool
```

Comment:
nit: similar to the renaming comment above, "blog" sounds confusing in the WooCommerce context
```suggestion
    @NSManaged public var isPublic: Bool
```

Reply:
I agree. Renamed in https://github.com/woocommerce/woocommerce-ios/pull/9480/commits/a8b9dd21252ce9faf6cb80f2e528f46e5040e3d8